### PR TITLE
New package: keybase-1.0.14

### PR DESCRIPTION
**Pull request by obosob**

### CODE REVIEW

File: srcpkgs/keybase/template [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Template file for 'keybase'
+pkgname=keybase
+short_desc="CLI tool for GPG with keybase.io"
+version=1.0.14
+revision=1
+wrksrc=client-${version}-${revision}
+build_style=go
+go_import_path=github.com/keybase/client/go/keybase
```
Comment (ebfe) [inline srcpkgs/keybase/template]:
```
go_import_path=github.com/keybase/client
go_package=github.com/keybase/client/go/keybase
```

File: srcpkgs/keybase/template [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+hostmakedepends="git"
+depends="gnupg"
+maintainer="Obosob <obosob@openmailbox.org>"
+license="BSD"
+homepage="https://keybase.io/"
+distfiles="https://github.com/keybase/client/archive/v${version}-${revision}.tar.gz"
+checksum=ef8f6d01206428a99558885ecec8de99089a60fe5f4b6e6f425e01a3fb368853
+
+pre_build(){
```
Comment (ebfe) [inline srcpkgs/keybase/template]: drop this function

File: srcpkgs/keybase/template [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  mkdir -p $GOPATH/src/github.com/keybase/
+  ln -sf ${wrksrc} $GOPATH/src/github.com/keybase/client
+  export GO15VENDOREXPERIMENT=1
+}
+post_install(){
+  vlicense LICENSE
```
Comment (ebfe) [inline srcpkgs/keybase/template]: indent with tabs

**Comment by obosob** [comment]
Fixed

File: srcpkgs/keybase/template [hunk 4]
```diff
@@ -0,0 +1,23 @@
+# Template file for 'keybase'
+pkgname=keybase
+short_desc="CLI tool for GPG with keybase.io"
+version=1.0.14
+revision=1
+wrksrc=client-${version}-${revision}
+build_style=go
+go_import_path=github.com/keybase/client
+go_package=github.com/keybase/client/go/keybase
+hostmakedepends="git"
```
Comment (ebfe) [inline srcpkgs/keybase/template]: not needed

File: srcpkgs/keybase/template [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+depends="gnupg"
+maintainer="Obosob <obosob@openmailbox.org>"
+license="BSD"
+homepage="https://keybase.io/"
+distfiles="https://github.com/keybase/client/archive/v${version}-${revision}.tar.gz"
+checksum=ef8f6d01206428a99558885ecec8de99089a60fe5f4b6e6f425e01a3fb368853
+
+pre_build() {
+	export GO15VENDOREXPERIMENT=1
```
Comment (ebfe) [inline srcpkgs/keybase/template]: again. drop this, this is the default with go 1.6.

File: srcpkgs/keybase/template [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+wrksrc=client-${version}-${revision}
+build_style=go
+go_import_path=github.com/keybase/client
+go_package=github.com/keybase/client/go/keybase
+hostmakedepends="git"
+depends="gnupg"
+maintainer="Obosob <obosob@openmailbox.org>"
+license="BSD"
+homepage="https://keybase.io/"
+distfiles="https://github.com/keybase/client/archive/v${version}-${revision}.tar.gz"
```
Comment (ebfe) [inline srcpkgs/keybase/template]: This is mixing upstream versioning with template revision. The upstream version is "1.0.14-1" which should be independent from revision.

File: srcpkgs/keybase/template [hunk 7]
```diff
@@ -0,0 +1,23 @@
+# Template file for 'keybase'
+pkgname=keybase
+short_desc="CLI tool for GPG with keybase.io"
+version=1.0.14
+revision=1
+wrksrc=client-${version}-${revision}
```
Comment (Vaelatern) [inline srcpkgs/keybase/template]: No. Revision is for our usage, not upstream.
Use:
```
revision=1
_version="${version%.*}-${version##*.}"
wrksrc="client-${_version}"
```
instead. It's cleaner. It also requires you to make `version=1.0.14.1` if that is indeed the version.

**Comment by Vaelatern** [comment]
As the package is created, it will not successfully run on a client system. I have an almost identical package in my WIP, it does not function properly.

**Comment by Vaelatern** [comment]
Changes were needed to the go build system. I have pushed a working copy of this package 3e84bbd, thank you for your help.

**Pull request closed without merging**